tools/gfx-unit-test/shared-buffers-tests.cpp

```diff
@@ -107,19 +107,19 @@ namespace gfx_test
             SLANG_IGNORE_TEST;
         }
 
-        sharedHandleTestImpl(srcDevice, dstDevice, context);
+        sharedBufferTestImpl(srcDevice, dstDevice, context);
     }
-#if SLANG_WIN64
-    SLANG_UNIT_TEST(sharedHandleD3D12ToCUDA)
+#if SLANG_WINDOWS_FAMILY
```

Comment:
This needs to be `SLANG_WIN64` since we can't run any cuda related stuff on x86.

Reply:
Fixed again